tests/unit/projection-test.js

```diff
@@ -955,7 +953,9 @@ module('unit/projection', function(hooks) {
       assert.equal(get(projectedExcerpt, 'author.age'), AUTHOR_AGE, 'excerpt has the correct author.age');
     });
 
-    test('Updating an embedded object property on a projection updates the base-record and other projections', function(assert) {
+    // Skipped because we cannot really simulate an update to a projection, it is always an update to the base record
+    // only fetches are something we can distinguish
+    skip('Updating an embedded object property on a projection updates the base-record and other projections', function(assert) {
```

Comment:
I would unskip this, we can simulate an update to a projection if `projectionTypes` is our way of signaling `partial-data`. The payload should be update however to push the empty-projection as the primary record and the partial book in `included`